teamsupport/models.py

```diff
@@ -23,14 +35,41 @@ def __init__(self, client, ticket_id=None, data=None):
                 '(neither given)')
         self.id = self.TicketID
 
+    @classmethod
+    def create(cls, first_name, email, category, name, description):
```

Comment:
That argument order is quite weird. Why is name so far from first_name? Is name the rest of the name? The entire name?